assets/admin_themes/Sea_Green/sea_green_variables.scss

```
@@ -23,9 +23,12 @@ $black        : #000;
 $black60      : #666666; // rgba(0, 0, 0, 0.6);
 $black75      : rgba(0, 0, 0, 0.75);
 $grape75 : rgba(123, 97, 255, 0.4);
+$grape90 : rgba(123, 97, 255, 0.9);
```

Stylelint (17.11.0)

[error] Unexpected whitespace before ":" (scss/dollar-variable-colon-space-before)

(scss/dollar-variable-colon-space-before)